src/pages/AboutUs/Team.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { MDBCol, MDBIcon } from 'mdbreact';
+import pic1 from './pic1.jpg';
+import pic2 from './pic2.jpg';
+import pic3 from './pic3.jpg';
+import pic4 from './pic4.jpg';
+import pic5 from './pic5.jpg';
+import { useTranslation } from 'react-i18next';
+import TeamMemberCards from './TeamMemberCards';
+
+function TeamMembersInformation() {
+  const { t } = useTranslation();
+  let team = [
+    {
+      member: {
+        name: t('aboutPage.saraAltaee'),
+        bio: t('aboutPage.saraBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: 'https://mdbootstrap.com/img/Photos/Avatars/img%20%2810%29.jpg',
+      },
+    },
+    {
+      member: {
+        name: t('aboutPage.neven'),
+        bio: t('aboutPage.nevenBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic1,
+      },
+    },
+    {
+      member: {
+        name: t('aboutPage.dunia'),
+        bio: t('aboutPage.duniaBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic2,
+      },
+    },
+    {
+      member: {
+        name: t('aboutPage.qays'),
+        bio: t('aboutPage.qaysBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic3,
+      },
+    },
+    {
+      member: {
+        name: t('aboutPage.zeena'),
+        bio: t('aboutPage.zeenaBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic4,
+      },
+    },
+    {
+      member: {
+        name: t('aboutPage.zainab'),
+        bio: t('aboutPage.zainabBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic5,
+      },
+    },
+  ];
```

Comment:
this logic is a bit overwhelming

try to fix the i18next above and that could give you some idea about how to improve this chunk of code